Tests/SentryTests/SentryHubTests.swift

```diff
@@ -67,8 +67,6 @@ class SentryHubTests: XCTestCase {
         fixture.fileManager.deleteAppState()
         fixture.fileManager.deleteTimestampLastInForeground()
         fixture.fileManager.deleteAllEnvelopes()
-        
-        sut = fixture.getSut()
```

Comment:
I tried running the new hub test case on main to see why it's crashing, because my mental model of this area of the SDK and the changes is not built up very much.

I noticed though that there were two live hub instances because this one is always set up; I made it lazy so it's not initialized for your new test case.